### DISCUSSION TITLE
Why GoAccess ignores TZ env variable?

### DISCUSSION THREAD

**Discussion by sbulatnikov** [allinurl/goaccess#2535] [Q&A]
I've tested several dozen setups with different implementations of TZ variable hoisting upside to GoAccess webserver. All of them were about some linux tricks and after all I made a conclusion that it TZ variable have no any effect on GoAccess webserver and especially on Last Updated field.

Can you explain without sending me to code reading(plz!!!) how does it work and how I can set correct TZ for GoAccess?

It's kinda interesting to get behaviour like this:
<img width="304" alt="image" src="https://github.com/allinurl/goaccess/assets/44680085/db96bee6-932a-4cd7-884b-ed3559a97ced">

**Comment by allinurl** [comment]
Are you using the latest version? Also, could you please share a few lines from your access log and the exact command you're using with GoAccess? Thanks

**Comment by allinurl** [comment]
Use GoAccess's `--tz` option with a canonical timezone, for example `--tz=Europe/Berlin` or `--tz=America/Chicago`. This also applies to the HTML "Last Updated" field. Relying only on the `TZ` environment variable can vary by container and installed timezone data. 

Closing this as stale, but feel free to reopen if `--tz` doesn't work with the latest release.
